src/safeds/ml/nn/layers/_dropout_layer.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from safeds._utils import _structural_hash
+from safeds._validation import _check_bounds, _OpenBound
+from safeds.ml.nn.typing import ModelImageSize
+
+from ._layer import Layer
+
+if TYPE_CHECKING:
+    from torch import nn
+
+
+class DropoutLayer(Layer):
+    """
+    Create a dropout Layer.
+
+    Parameters
+    ----------
+    probability:
+        The probability of which the input neuron becomes an output neuron
+
+    Raises
+    ------
+    OutOfBoundsError
+        If probability < 0
+        If probability > 1
+    """
+
+    def __init__(self, probability: float):
+        _check_bounds("probability", probability, lower_bound=_OpenBound(0), upper_bound=_OpenBound(1))
+        self.probability = probability
```

Comment:
```suggestion
        self._probability = probability
```

Fields should be internal. Add a property method called `probability` to access the value.